values/vector.go

```diff
@@ -0,0 +1,13 @@
+package values
+
+import (
+	"github.com/influxdata/flux/semantic"
+)
+
+type Vector interface {
+	Value
+	ElementType() semantic.MonoType
+	Get(i int) Value
```

Comment:
I think you can just remove most of these attributes outside of `ElementType()`. Instead, this should expose the arrow array. We're not going to use these methods anyway.

Reply:
> I think you can just remove most of these attributes outside of `ElementType()`.

Including the `Value` mixin?

Reply:
The `Value` mixin is needed for the values interface so keep that. But, we should expose the array as much as possible since that's what we'll actually use. Access methods outside of that won't be helpful.